main/EDTypesMod.F90

```diff
@@ -189,7 +189,7 @@ module EDTypesMod
                                                         ! in PRTGenericMod.F90. examples are carbon12_element
                                                         ! nitrogen_element, etc.
 
-  integer, public :: element_pos(num_organ_types)       ! This is the reverse lookup
+  integer, public :: element_pos(num_element_types)       ! This is the reverse lookup
```

Comment:
I see `num_organ_types` and `num_element_types` are defined to have the same value currently.  I'm guessing `num_organ_types` was the misapplied originally?  Or was it something else I'm missing?

Reply:
yes, they just happen to be the same size
It was originally a typo, the number of element positions should have the same size as the number of elements.